ethereum_history_api/circuits/lib/src/storage.nr

```diff
@@ -0,0 +1,14 @@
+use crate::misc::types::Bytes32;
+use crate::account::PROOF_LEN;
+
+struct StorageProofsWithStorageRoot<N> {
+    storage_root: Bytes32,
+    proofs: [StorageProof; N]
+}
+
+struct StorageProof {
+    key: Bytes32,
+    value: Bytes32,
+    proof: [u8; PROOF_LEN],
```

Comment:
We should not reuse the const from account here. It's computed based on the max number of levels which would be smaller here. Max level length should stay the same.

Also - while we are at it - please change the const there. Now it's :
```rust
global PROOF_LEN = 4788;
```
Which is unclear - where does it come from. It's MAX_TRIE_NODE_LENGTH * 9 where 9 is MAX_ACCOUNT_PROOF_LEVELS. We should have a similar const here but with storage proof levels.

Reply:
👍 we should have `STATE_PROOF_LEN` and `STORAGE_PROOF_LEN`

Reply:
Two proof lengths are now separated